lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -236,23 +236,41 @@ function paraneue_dosomething_preprocess_reportback_gallery(&$vars, $count = 6)
     'random' => TRUE,
   ];
 
+  // If the Rogue feature flag is on, grab reportbacks from Rogue.
+  if (variable_get('rogue_collection', FALSE)) {
+    // Hit the GET /reportbacks endpoint in Rogue with campaign_id filter.
+    // On the Rogue side, it will only return 'approved', which will be shown in the gallery.
+    $promotedReportbackItems = dosomething_rogue_get_reportback_items(['campaign_id' => $gallery['campaign_id']])['data'];
```

Comment:
I think it's safe to only grab a maximum of 6 here since everything else is fetched when hitting the "view more" button in the user's browser (which will go directly to Rogue). If you need a count of total accepted (below), you can probably grab that from the `meta` on this response.